specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/gallery.json

```diff
@@ -40,7 +40,7 @@
           "Galleries"
         ],
         "operationId": "Galleries_CreateOrUpdate",
-        "description": "Create or update a gallery.",
+        "description": "Create or update a Shared Image Gallery.",
```

Comment:
all changes in this file are already there in Azure master branch.. Not sure why its showing as "added" in my PR, as I simply pulled Azure master into my forked repository.